frontend/public/components/dashboard/top-consumers-card/consumers-filter.tsx

```diff
@@ -4,18 +4,23 @@ import { CPU_DESC, MEMORY_DESC, STORAGE_DESC, NETWORK_DESC } from './strings';
 import { Humanize, humanizeBinaryBytesWithoutB, humanizeDecimalBytesPerSec, humanizeSeconds } from '../../utils';
 import { MetricType } from './metric-type';
 
+const toNanoSeconds = (value: React.ReactText) => {
```

Comment:
Nit: Probably better to write this type as `string | number` since these humanize functions are generally not React specific.